Mock_Up/ViewController.swift

```diff
@@ -28,6 +29,25 @@ class ViewController: UIViewController, UINavigationControllerDelegate, UIImageP
     
     
     
+    @IBAction func usePhoto(_ sender: Any) {
+        // pass image to second view controller
+        self.theImage = myImageview.image!
+        performSegue(withIdentifier: "name", sender: self)
+    }
+    
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
```

Comment:
Before any segue happens (after the photo is taken), it will call the second instance of the view controller and pass the new photo over